app/src/main/res/layout/pin_entry_card.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge
+  tools:parentTag="android.support.v7.widget.CardView"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <ProgressBar
+    android:id="@+id/pinentry_progress"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:visibility="invisible" />
+
+  <LinearLayout
+    android:id="@+id/pinentry_pin_container"
+    style="@style/Clinic.CardContentContainer.AppLock">
+
+    <TextView
+      style="@style/Clinic.LoginPinLabel"
+      android:labelFor="@+id/pinentry_pin"
+      android:text="@string/loginpin_pin_number_hint" />
```

Comment:
Can we also rename the string resource `loginpin_pin_number_hint` to be more specific to PinCardView?

Reply:
Yes, thanks for catching this x2.

Reply:
Adding `@string/pinentry_pin_number_hint`.